### ISSUE TITLE
Add qualifying crimes notifier

### ISSUE THREAD

**Issue by scottfvt** [codeforbtv/expunge-vt#46]
Add notifier to popup indicating the crime is a qualifying crime. This would be in line with the other notifiers for each crime.

**Comment by jeremybrault** [comment]
https://vtlawhelp.org/expungement < what is expungeable

**Comment by jeremybrault** [comment]
https://legislature.vermont.gov/statutes/section/13/230/07601 < statue that provides more detail about what is and is not expungeable.

**Comment by jeremybrault** [comment]
We are going to compile a spreadsheet of all expungeable counts... including their title, their section number, and offense class. Then this will be checked against in the UI to provide a default filing option of "Ineligible" or "Expunge Conviction/NonConviction"...etc.

**Comment by jeremybrault** [comment]
@jakedurell Is this the same as: https://github.com/codeforbtv/expunge-vt/issues/16?

**Comment by jakedurell** [comment]
Yeah @jeremybrault , this is redundant. I think we should put this on hold for now. Will close this one.
